pyccel/ast/macros.py

```diff
@@ -148,10 +148,6 @@ class MacroCount(Macro):
     def __str__(self):
         return f'MacroCount({self.argument})'
 
-
-
-
-
 def construct_macro(name, argument, parameter=None):
```

Comment:
Beware of unnecessary changes in PRs that just make reviews longer

Reply:
oky, I will be careful next time Thanks